service/routes.py

```diff
@@ -29,6 +29,22 @@ class filter_type(Enum):
 # end enum filter_type
 
 
+class update_status_type(Enum):
+    """ Enumeration of update status types """
+
+    DISABLED = (0,)
+    ENABLED = (1,)
+
+    # This is the last value in the enum and is meant to be a default value. Nothing should come after this
+    UNKNOWN = 2
+
+
+# end enum update_status_type
+
+# Global table where each tuple is a {(str Product ID, enum Condition), enum update_status_type} pair
+status_dict = {}
```

Comment:
This seems like it would partially mirror the database no?
Would it make sense to just add an isUpdateEnabled attribute to our database that is only updated by us?
And then we can make a utility function that is called in the Update routes to check if updates are enabled or disabled?
(I can add the attribute to our model in a diff PR if you want me to btw)